dask/dataframe/core.py

```diff
@@ -4634,13 +4702,6 @@ def set_index(
         >>> ddf2 = ddf.set_index("x")
         >>> ddf2 = ddf.set_index(ddf.x)
         >>> ddf2 = ddf.set_index(ddf.timestamp, sorted=True)
-
-        A common case is when we have a datetime column that we know to be
-        sorted and is cleanly divided by day.  We can set this index for free
-        by specifying both that the column is pre-sorted and the particular
-        divisions along which is is separated
-
-        >>> import pandas as pd
```

Comment:
This information still looks useful here, any reason to remove it? I didn't see a note in your PR summary.

Reply:
Actually I don't know why that was erased. I'll check that out.

Reply:
This no longer appears in [core.py](https://github.com/dask/dask/blob/main/dask/dataframe/core.py, I guess it was removed by some previous merge?

Reply:
This should've been corrected now. Apologies!